scripts/us_usda/quickstats/process.py

```diff
@@ -0,0 +1,172 @@
+import requests
+import sys
+import csv
+import multiprocessing
+from itertools import repeat
+import os
+import datetime
+
+API_BASE = 'https://quickstats.nass.usda.gov/api'
+
+API_KEY = '569256A0-9CF4-34F0-86F3-FC477003330A'
```

Comment:
We could move the key to a config file on GCS so it doesn't get blocke din the future.

Reply:
Done.